demos/smartlab_online_demo/python/object_detection/data/clsdata_io.py

```diff
@@ -0,0 +1,34 @@
+"""
+ Copyright (C) 2021-2022 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# folder2imagenet, imagenet2folder
+# split_labels, get_subset_labels
+
+# augment:
+  # enumerate class folders
+  # augment images to reach the number: N
+  # folder2imagenet
+    # dir - ....jpg
+    #     - labels.txt
+    #     - classes.txt
+  # split_labels
+# debug:
+  # get_subset_labels
+    # dir - ....jpg
+    #     - sub-labels.txt
+    #     - classes.txt
+  # imagenet2folder
+  # folder2imagenet
```

Comment:
This file can be removed

Reply:
thanks for the suggestion. remove the file